### ISSUE TITLE
ionic --version outputs non readable characters

### ISSUE THREAD

**Issue by metashock** [ionic-team/ionic#12300]
**Ionic version:**  (check one with "x")
[ ] **1.x** (For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[ x] **3.x**

**I'm submitting a ...**  (check one with "x")
[x ] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
ionic --version outputs non readable characters along with the version number making it hard to parse with other tools.

**Expected behavior:**
Output just the version number as text.

**Steps to reproduce:**

    ionic --version | cat -A
    ^[[2K^[[G3.4.0$

**Ionic info:** 

```
root@294ca6e0a4d9:/node-v6.11.0-linux-x64# ionic info              
[WARN] You are not in an Ionic project directory. Project context may be missing.

global packages:

    @ionic/cli-utils : 1.4.0
    Ionic CLI        : 3.4.0

System:

    Node       : v6.11.0
    OS         : Linux 4.8
    Xcode      : not installed
    ios-deploy : not installed
    ios-sim    : not installed
    npm        : 3.10.10 


```

**Comment by jgw96** [comment]
This issue was moved to ionic-team/ionic-cli#2507
